refresh_test.py

```diff
@@ -145,3 +145,30 @@ def test_refresh_node(self):
         self.end = self.get_current_timestamp()
 
         self.check_timeout()
+
+    def test_node_recovery(self):
+
+        self.log.info('DB cluster is: %s', self.db_cluster)
+        assert len(self.db_cluster.nodes) == 2, "The test requires 2 DB nodes!"
+        stress_queue = self.run_stress_thread(stress_cmd=self.params.get('stress_cmd'))
+        self.get_stress_results(queue=stress_queue)
+        self.verify_stress_thread(cs_thread_pool=stress_queue)
+
+        # stop all nodes
+        nodes = self.db_cluster.nodes
+        for node in nodes:
+            kill_cmd = "sudo pkill -9 scylla"
+            node.remoter.run(kill_cmd, ignore_status=True)
+            node.wait_db_up(timeout=14400)
+            node.wait_jmx_up()
```

Comment:
In this case what will happen is rolling stop and not stop of all nodes in the same time.

Reply:
we need to disable the service so it won't start after killing it, then kills all nodes.
Then, verify all nodes are down.
Then, start all nodes.